index.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Home Page</title>
+</head>
+<body>
+  <div class="grid-container">
+    <nav class="navbar">
+      <div class="hamburger small-screen">
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
```

Comment:
Please ensure that your `navbar` is like the original design.  Please ensure to add the top short bar section with social media links, logout, and language links aligned to the right.

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/163654675-1f601f5c-77e9-4fb0-93c3-ad83198dec45.png) | ![image](https://user-images.githubusercontent.com/10332499/163654634-34d3c3b4-87d3-4799-9444-9b333c65d5ee.png)